.github/workflows/turf.yml

```diff
@@ -38,3 +38,53 @@ jobs:
       - run: yarn --frozen-lockfile
       - run: git diff --exit-code
       - run: yarn test
+
+  coverage:
```

Comment:
Coverage winds up taking a long time because it has to do all of the builds in the `yarn install` step. Circle has the ability to save a workspace and then fork into multiple other tasks. Maybe we can do that here and spend a lot less time on the coverage step? Otherwise we could just calculate it as part of the regular builds since they run tests anyhow.

Also having to re-run coverage on master for every PR seems like its just burning CPU. Maybe we can store the results somewhere and use that for the diff instead of needing to recalculate them every time. I'm not actually sure how people do this in other repos.

Reply:
I'm not super concerned here because the job processes run in parallel - they shouldn't impact each others performance. 

I've removed `master` for now, better to just get coverage working and then figure out how to do a diff against master.